### PR TITLE
Adds the debug option to thumbor urls

### PR DESCRIPTION
**Pull request by okor**
It's sometimes super helpful to have the debug option, when developing application level cropping logic. So I suggest adding it here.

### CODE REVIEW

**Comment by okor** [comment]
That failure https://travis-ci.org/thumbor/ruby-thumbor/builds/121544114 is only in Ruby head, related to installing ffi. So that has nothing to do with this PR.

**Comment by guilhermef** [comment]
@okor, now it's passing, but you have to create a test for the debug url

File: lib/thumbor/crypto_url.rb [hunk 1]
```diff
@@ -132,6 +132,10 @@ def url_for(options, include_hash = true)
 
             url_parts = Array.new
 
+            if options[:debug]
+                url_parts.push('debug')
```
Comment (guilhermef) [inline lib/thumbor/crypto_url.rb]: This can be written as:
```
url_parts.push('debug') if options[:debug]
```

**Comment by okor** [comment]
@guilhermef my bad :blush: will do

**Pull request merged**